library/src/main/java/com/getstream/sdk/chat/adapter/viewholder/message/configurators/MessageTextConfigurator.kt

```diff
@@ -0,0 +1,201 @@
+package com.getstream.sdk.chat.adapter.viewholder.message.configurators
+
+import android.content.Context
+import android.graphics.Color
+import android.graphics.drawable.Drawable
+import android.text.SpannableString
+import android.text.SpannableStringBuilder
+import android.text.style.ForegroundColorSpan
+import android.text.style.RelativeSizeSpan
+import android.util.TypedValue
+import android.widget.TextView
+import androidx.constraintlayout.widget.ConstraintLayout
+import androidx.core.content.ContextCompat
+import androidx.core.view.isGone
+import androidx.core.view.isVisible
+import androidx.core.view.updateLayoutParams
+import com.getstream.sdk.chat.Chat
+import com.getstream.sdk.chat.R
+import com.getstream.sdk.chat.adapter.MessageListItem.MessageItem
+import com.getstream.sdk.chat.adapter.MessageViewHolderFactory
+import com.getstream.sdk.chat.adapter.viewholder.message.isDeleted
+import com.getstream.sdk.chat.adapter.viewholder.message.isFailed
+import com.getstream.sdk.chat.databinding.StreamItemMessageBinding
+import com.getstream.sdk.chat.navigation.destinations.WebLinkDestination
+import com.getstream.sdk.chat.utils.StringUtility
+import com.getstream.sdk.chat.utils.Utils
+import com.getstream.sdk.chat.view.MessageListView
+import com.getstream.sdk.chat.view.MessageListViewStyle
+import io.getstream.chat.android.client.ChatClient
+import io.getstream.chat.android.client.models.Message
+import java.util.Arrays
+
+internal class MessageTextConfigurator(
+    private val binding: StreamItemMessageBinding,
+    private val context: Context,
+    private val style: MessageListViewStyle,
+    private val bubbleHelper: MessageListView.BubbleHelper,
+    private val messageClickListener: MessageListView.MessageClickListener,
+    private val messageLongClickListener: MessageListView.MessageLongClickListener
+) : Configurator {
+
+    override fun configure(
+        messageItem: MessageItem
+    ) {
+        val message = messageItem.message
+
+        if (message.text.isEmpty() && !message.isDeleted()) {
+            binding.tvText.isVisible = false
+            return
```

Comment:
How about using else branch instead of return in the middle?

Reply:
I'm actually a fan of doing early returns like this, as it avoids the nesting that the `else` branch would introduce.

I think it's quite clear here that if the check fails then we don't have anything to display, so we're just hiding the View and skipping the rest of the operations.